final_task/rss_parser/Output_functions.py

```diff
@@ -0,0 +1,227 @@
+import json
+import logging
+import re
+from itertools import groupby
+
+from bs4 import BeautifulSoup
+
+from final_task.rss_parser.Classes.Novelty import Novelty
+
+
+def clean_html(raw_html):
+    """
+    Formatting our String excluding all html-tags making it easier to read our description
+    :param raw_html:
+    :return:
+    """
+    logging.info("Cleaning something from htlm-tags.")
+    cleaner = re.compile('<.*?>')
+    clean_text = re.sub(cleaner, '', raw_html)
+    logging.info("Cleaned from html-tags!")
+    return clean_text
+
+
+def printing_corrected(the_feed_entry):
```

Comment:
Python have a lot of formatting tools. You may want to use one of them instead of this function

Reply:
Corrected.